test/__dom_fixtures__/insertChildren/output.js

```diff
@@ -1,19 +1,92 @@
-'use strict';
+import { template as _$template } from "r-dom";
+import { spread as _$spread } from "r-dom";
+import { insert as _$insert } from "r-dom";
+import { createComponent as _$createComponent } from "r-dom";
 
-var _dom = require('dom');
+const _tmpl$ = _$template(`<div></div>`),
+  _tmpl$2 = _$template(`<module></module>`),
+  _tmpl$3 = _$template(`<module>Hello</module>`);
 
-const _tmpl$ = (0, _dom.template)(`<div></div>`),
-  _tmpl$2 = (0, _dom.template)(`<module></module>`);
+const children = _tmpl$.cloneNode(true);
 
-var child = _tmpl$.cloneNode(true);
+const dynamic = {
+  children
+};
 
-var template = (0, _dom.createComponent)(Module, {
-  children: child,
+const template = _$createComponent(Module, {
+  children: children
 });
 
-var template2 = (function() {
-  var _el$2 = _tmpl$2.cloneNode(true);
+const template2 = (function() {
+  const _el$2 = _tmpl$2.cloneNode(true);
+
+  _$insert(_el$2, children);
 
-  (0, _dom.insert)(_el$2, child);
   return _el$2;
 })();
+
+const template3 = _tmpl$3.cloneNode(true);
+
+const template4 = (function() {
+  const _el$4 = _tmpl$2.cloneNode(true);
+
+  _$insert(_el$4, _$createComponent(Hello, {}));
+
+  return _el$4;
+})();
+
+const template5 = (function() {
+  const _el$5 = _tmpl$2.cloneNode(true);
+
+  _$insert(_el$5, () => dynamic.children);
+
+  return _el$5;
+})();
+
+const template6 = _$createComponent(
+  Module,
+  {
+    children: () => dynamic.children
+  },
+  ["children"]
+);
+
+const template7 = (function() {
+  const _el$6 = _tmpl$2.cloneNode(true);
+
+  _$spread(_el$6, dynamic, false);
+
+  return _el$6;
+})();
+
+const template8 = (function() {
+  const _el$7 = _tmpl$3.cloneNode(true);
+
+  _$spread(_el$7, dynamic, false, true); // ignore children
```

Comment:
@ryansolid 
Do you think ignore children flag is the best way?

Reply:
I was wrong. Looks like actual children win in React.